homeassistant/components/alexa/handlers.py

```diff
@@ -1523,3 +1524,30 @@ def temperature_from_object(hass, temp_obj, interval=False):
     )
 
     return directive.response()
+
+
+@HANDLERS.register(("Alexa.CameraStreamController", "InitializeCameraStreams"))
+async def async_api_initialize_camera_stream(hass, config, directive, context):
+    """Process a InitializeCameraStreams request."""
+    entity = directive.entity
+    camera_component = hass.data.get(camera.DOMAIN)
```

Comment:
This is not allowed. We are not allowed to access `hass.data` from other domains.

Reply:
We need to add a new helper to the camera integration like `async_request_stream` if we want to get a snapshot url.

Reply:
Added `async_get_image_url()` helper to `camera`.